ignite/src/test/java/org/hibernate/ogm/datastore/ignite/test/queries/QueriesWithAssociationsTest.java

```diff
@@ -0,0 +1,193 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.ignite.test.queries;
+
+import static org.fest.assertions.Assertions.assertThat;
+
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.List;
+import java.util.TimeZone;
+
+import javax.persistence.EntityManager;
+
+import org.hibernate.ogm.backendtck.queries.Address;
+import org.hibernate.ogm.backendtck.queries.Author;
+import org.hibernate.ogm.backendtck.queries.Hypothesis;
+import org.hibernate.ogm.utils.jpa.OgmJpaTestCase;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * Copied from {@link org.hibernate.ogm.backendtck.queries.QueriesWithAssociationsTest}
+ * but Ignite queries are not transactional so we need another algorithm in {@code removeEntities()}
+ *
+ * @author Victor Kadachigov
+ */
+public class QueriesWithAssociationsTest extends OgmJpaTestCase {
```

Comment:
Is this test related to the MultiGet implementation?

Reply:
No.

Reply:
Test org.hibernate.ogm.backendtck.queries.QueriesWithAssociationsTest have fixed already. I will send it in next PR. Could you please remove the test?

Reply:
Why? I think it's correct. When you make your PR you will be able make to changes